### PR TITLE
Remove some unused packages

### PR DESCRIPTION
**Pull request by janpaul123**
And add some that were missing. And also made it so docs open in a
browser just like storybook.

Test plan: covered by CI, manually checked storybook and docs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,15 +5,11 @@
     "@babel/cli": "^7.1.5",
     "@babel/core": "^7.2.0",
     "@babel/plugin-proposal-class-properties": "^7.2.1",
-    "@babel/plugin-proposal-export-default-from": "^7.0.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.2.0",
```
Comment (vidaaudrey) [inline package.json]: we use `transform-runtime` in rollup.config.js

File: package.json [hunk 2]
```diff
@@ -5,15 +5,11 @@
...
-    "@babel/preset-env": "^7.1.6",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@mdx-js/loader": "^0.16.6",
-    "@mdx-js/mdx": "^0.16.6",
```
Comment (vidaaudrey) [inline package.json]: we are still using `mdx-js` loader in webpack.config.js right?

File: package.json [hunk 3]
```diff
@@ -23,12 +19,8 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^23.4.2",
     "babel-loader": "^8.0.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "babel-preset-flow": "^6.23.0",
```
Comment (vidaaudrey) [inline package.json]: we use it in babel.config.js

File: package.json (hunk 1 above)
Comment (janpaul123) [reply package.json]: Ah shoot I totally missed that. For some reason the build still works without failing. 🤔

File: package.json (hunk 2 above)
Comment (janpaul123) [reply package.json]: Yep!

File: package.json (hunk 3 above)
Comment (janpaul123) [reply package.json]: I think that one is using `@babel/preset-flow`.

**Comment by janpaul123** [comment]
> eslint can't resolve the regl-worldview module now, we'll need to add the package path to import resolver kriasoft/react-starter-kit#1180

Hm the webviz resolver should take care of that automatically. I'll dig into that.

File: package.json (hunk 3 above)
Comment (vidaaudrey) [reply package.json]: oh, you are right. forgot about it

**Pull request merged**